docs/advanced-features/athena.ipynb

```diff
@@ -0,0 +1,688 @@
+    "top hashes available in Athena.\n",
+    "<!--pytest-codeblocks:cont-->"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "1ae3532c-a91e-4168-824a-3a73a941f9c6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "DDL[PACKAGES_TABLE] = f\"\"\"\n",
+    "CREATE EXTERNAL TABLE IF NOT EXISTS `{ATHENA_DB}.{PACKAGES_TABLE}`(\n",
+    "  `hash` string)\n",
+    "ROW FORMAT DELIMITED\n",
+    "  FIELDS TERMINATED BY ',' STORED AS INPUTFORMAT\n",
+    "  'org.apache.hadoop.mapred.TextInputFormat'\n",
+    "OUTPUTFORMAT\n",
+    "  'org.apache.hadoop.hive.ql.io.HiveIgnoreKeyTextOutputFormat'\n",
+    "LOCATION\n",
+    "  '{QUILT_URL}/.quilt/named_packages'\n",
+    "TBLPROPERTIES (\n",
+    "  'has_encrypted_data'='false',\n",
+    "  'transient_lastDdlTime'='1557626200')\n",
+    "\"\"\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "aa42355a-6791-4a6c-902e-29b64943abef",
+   "metadata": {},
+   "source": [
+    "### C. View of package-level metadata\n",
+    "The DDL below creates a view that contains package-level information including: \n",
+    "* User\n",
+    "* Package name\n",
+    "* Tophash\n",
+    "* Timestamp\n",
+    "* Commit message\n",
+    "<!--pytest-codeblocks:cont-->"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "d8d48a9c-62a0-46d0-960a-ba938d81644d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "SLASH=r'\\/([^\\/]+)'\n",
+    "S1=r'\\/'\n",
+    "S3_MATCH=f'^s3:{S1}{SLASH}{SLASH}{SLASH}{SLASH}'\n",
+    "DDL[PACKAGES_VIEW] = f\"\"\"\n",
+    "CREATE OR REPLACE VIEW {ATHENA_DB}.{PACKAGES_VIEW} AS\n",
+    "WITH\n",
+    "  npv AS (\n",
+    "    SELECT\n",
+    "      regexp_extract(\"$path\", '{S3_MATCH}', 4) as user,\n",
+    "      regexp_extract(\"$path\", '{S3_MATCH}{SLASH}', 5) as name,\n",
+    "      regexp_extract(\"$path\", '[^/]+$') as timestamp,\n",
+    "      {PACKAGES_TABLE}.\"hash\"\n",
+    "      FROM {PACKAGES_TABLE}\n",
+    "  ),\n",
+    "  mv AS (\n",
+    "    SELECT\n",
+    "      regexp_extract(\"$path\", '[^/]+$') as tophash,\n",
+    "        manifest.\"meta\",\n",
+    "        manifest.\"message\"\n",
+    "      FROM\n",
+    "        {MANIFEST_TABLE} as manifest\n",
+    "      WHERE manifest.\"logical_key\" IS NULL\n",
+    "  )\n",
+    "SELECT\n",
+    "  npv.\"user\",\n",
+    "  npv.\"name\",\n",
+    "  npv.\"hash\",\n",
+    "  npv.\"timestamp\",\n",
+    "  mv.\"message\",\n",
+    "  mv.\"meta\"\n",
+    "FROM npv\n",
+    "JOIN\n",
+    "  mv\n",
+    "ON\n",
+    "  npv.\"hash\" = mv.\"tophash\"\n",
+    "\"\"\"\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2111bfa0-ccea-4169-aeda-b67b8b317ed9",
+   "metadata": {},
+   "source": [
+    "### D. View of object-Level metadata\n",
+    "The DDL below creates a view that contains package contents, including:\n",
+    "* logical_key\n",
+    "* physical_keys\n",
+    "* object hash\n",
+    "* object metadata\n",
```

Comment:
I think we should use consistent styling of items between these ones and ones from the previous section (casing, using of space or underscore).

Reply:
Agreed